### ISSUE TITLE
Bitcode issue when compiling Xcode 7 beta, iOS 9

### ISSUE THREAD

**Issue by cannyboy** [0xced/XCDYouTubeKit#165]
I get this error when compiling, using Xcode 7 beta 4, for iOS 9 (for Device only, simulator is fine):

```
Ld /Users/cannyboy/Library/Developer/Xcode/DerivedData/MyApp-empxijxnnkgsnidrbjehcdiftlzj/Build/Intermediates/Pods.build/Debug-iphoneos/XCDYouTubeKit.build/Objects-normal/armv7/XCDYouTubeKit normal armv7
    cd /Users/cannyboy/Source-Files/App/MyApp-5/Pods
    export IPHONEOS_DEPLOYMENT_TARGET=9.0
    export PATH="/Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode-beta.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch armv7 -dynamiclib -isysroot /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.0.sdk -L/Users/cannyboy/Library/Developer/Xcode/DerivedData/MyApp-empxijxnnkgsnidrbjehcdiftlzj/Build/Products/Debug-iphoneos -F/Users/cannyboy/Library/Developer/Xcode/DerivedData/MyApp-empxijxnnkgsnidrbjehcdiftlzj/Build/Products/Debug-iphoneos -filelist /Users/cannyboy/Library/Developer/Xcode/DerivedData/MyApp-empxijxnnkgsnidrbjehcdiftlzj/Build/Intermediates/Pods.build/Debug-iphoneos/XCDYouTubeKit.build/Objects-normal/armv7/XCDYouTubeKit.LinkFileList -install_name @rpath/XCDYouTubeKit.framework/XCDYouTubeKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -miphoneos-version-min=9.0 -dead_strip -fembed-bitcode-marker -Wl,-U,_JSContextGetGlobalObject -Wl,-U,_JSEvaluateScript -Wl,-U,_JSGlobalContextCreate -Wl,-U,_JSGlobalContextRelease -Wl,-U,_JSObjectCallAsFunction -Wl,-U,_JSObjectIsFunction -Wl,-U,_JSObjectMake -Wl,-U,_JSObjectSetProperty -Wl,-U,_JSStringCopyCFString -Wl,-U,_JSStringCreateWithCFString -Wl,-U,_JSStringRelease -Wl,-U,_JSValueIsObject -Wl,-U,_JSValueIsString -Wl,-U,_JSValueMakeString -Wl,-U,_JSValueToStringCopy -framework MediaPlayer -fobjc-arc -fobjc-link-runtime -framework Foundation -framework MediaPlayer -single_module -compatibility_version 2 -current_version 2.2.0 -Xlinker -dependency_info -Xlinker /Users/cannyboy/Library/Developer/Xcode/DerivedData/MyApp-empxijxnnkgsnidrbjehcdiftlzj/Build/Intermediates/Pods.build/Debug-iphoneos/XCDYouTubeKit.build/Objects-normal/armv7/XCDYouTubeKit_dependency_info.dat -o /Users/cannyboy/Library/Developer/Xcode/DerivedData/MyApp-empxijxnnkgsnidrbjehcdiftlzj/Build/Intermediates/Pods.build/Debug-iphoneos/XCDYouTubeKit.build/Objects-normal/armv7/XCDYouTubeKit

ld: -U and -bitcode_bundle (Xcode setting ENABLE_BITCODE=YES) cannot be used together
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

XCDYouTubeKit version: 2.2.0. 

My app's project has this setting: `ENABLE_BITCODE = NO`, and XCDYouTubeKit makes no mention of bitcode, so not sure what the problem is.

**Comment by 0xced** [comment]
Can you please try with the latest version of CocoaPods (0.38.2) and the following entry in your Podfile?

    pod 'XCDYouTubeKit', { :git => 'https://github.com/0xced/XCDYouTubeKit.git', :commit => '5e45d727b148012ae889491b6ee78e3cf9b5b22a' }

I think it may fix your issue.

**Comment by cannyboy** [comment]
Thanks, this has fixed the issue.

**Comment by 0xced** [comment]
Great, I’ll release version 2.2.1 soon.

**Comment by 0xced** [comment]
For reference: commit 5e45d727b148012ae889491b6ee78e3cf9b5b22a fixed this issue.

**Comment by harryworld** [comment]
+1

**Comment by kevincador** [comment]
Do you have an ETA for the release of the version 2.2.1 on CocoaPods?

**Comment by 0xced** [comment]
I hope to release it during this week.

**Comment by 0xced** [comment]
I just released version 2.3.0, bitcode will work fine.
